modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,76 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install logging add-on services on {product-title}
+
+{product-tite} (ROSA) does not come preconfigured with a logging solution, but logging can be enabled by manually installing the `cluster-logging-operator` service.
+
+Log forwarding capability is provided through the Amazon CloudWatch add-on service.
+
+.Prerequisites
+
+Request the following link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[quotas from AWS]:
+* Quota for the `cluster-logging-operator` add-on
+* Quota for the Amazon CloudWatch add-on
+
+.Procedure
+
+To install log forwarding using interactive mode:
+
+. Enter the following command:
++
+[source,terminal]
+----
+$ rosa install addon cluster-logging-operator --cluster=<cluster_name> --interactive
+----
++
+For `<cluster_name>`, enter the name of your cluster.
+
+. When prompted, enter `yes` to install `cluster-logging-operator`.
+. When prompted, enter `yes` to install the optional Amazon CloudWatch log forwarding add-on.
+. Enter `yes` or `no` for the log type prompts: applications, infrastructure, and audit logs.
++
+*Applications logs*
```

Comment:
I don't know how this will look once it builds in a real doc, but I'd consider maybe putting these in bullets with a colon and then the description after it or something.

Reply:
It should build like a definition list as is, but without a preview, who knows. I'm accepting your suggestion. Thanks!